Task7/.env

```diff
@@ -0,0 +1,5 @@
+DB_HOST = localhost
```

Comment:
never commit `.env` , why are you committing database credentials to public repo ?